packages/image_picker/image_picker_android/android/gradle.properties

```diff
@@ -12,3 +12,4 @@
 # org.gradle.parallel=true
 #Fri Jan 27 08:52:19 CST 2023
 org.gradle.jvmargs=-Xmx1536M -Dkotlin.daemon.jvm.options\="-Xmx1536M"
+android.useAndroidX=true
```

Comment:
Do we really need this? I thought it was app-level rather than library-level.

Reply:
I had some tests failing due to this. I'll see if that is still the case.